src/Components/Search/SearchBarInput.tsx

```diff
@@ -66,16 +66,16 @@ const SearchBarInput: FC<SearchBarInputProps> = ({
   const formattedOptions: SuggestionItemOptionProps[] = [
```

Comment:
This is one of the more obvious tells that something could be simpler + safer with fragment containers: some array `formattedFoo`